babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+      '@babel/preset-env',
+      '@babel/preset-react',
+    ],
+  };  
```

Comment:
I am wondering can't the presets added directly to the `package.json`? instead of separate file.

Example here
https://babeljs.io/docs/configuration

Reply:
Didn't realize that was an option when seeing install examples online. added to package.json